templates/definition/charger/viessmann.yaml

```diff
@@ -75,40 +82,15 @@ render: |
     # false -> oneTimeCharge is disabled -> normal mode -> 1
     # true -> oneTimeCharge is enabled -> boost mode -> 2
   setmode:
-    source: switch
-    switch:
-    - case: 1 # normal
-      set:
-        source: http
-        uri: https://api.viessmann.com/iot/v2/features/installations/{{.installation_id}}/gateways/{{.gateway_serial}}/devices/{{.device_id}}/features/heating.dhw.oneTimeCharge/commands/deactivate
-        method: POST 
-        headers:  
-          - content-type: application/json
-        auth:
-          source: viessmann
-          user: {{ .user }}
-          password: {{ .password }}
-          clientid: {{ .clientid }}
-        body: >
-          { }
-    - case: 2 # boost
-      set:
-        source: sequence
+    source: watchdog
+    timeout: {{ .watchdog }} # re-write at timeout/2
```

Comment:
This needs to reset on mode 1

Reply:
Oh nvm, I think I just found it. Can you take a look please?